src/Tribe/Tickets.php

```diff
@@ -33,6 +33,16 @@ class Tribe__Tickets__Tickets {
 		 */
 		private static $have_displayed_reg_link = false;
 
+		/**
+		 * Function that is used to store the cache of a specific post associated with a set of tickets, where %d is the
+		 * ID of the post being affected.
+		 *
+		 * @since TBD
+		 *
+		 * @var string
+		 */
+		private static $cache_key_all_tickets = 'tribe_event_tickets_from_%d';
```

Comment:
if this was `$cache_key_prefix` you could avoid all the `sprintf` calls later and just concatenate

Reply:
also, making this variable `static` is a surprising choice. Since it is private, you can't access it outside the object anyway. There are other variables surrounding it which _are_ static, so, perhaps this was just pattern matching with the rest?

Reply:
1. I think having the variable as a prefix makes more sense thanks.
2. `private` to prevent inherit it down any child class and `static` just to follow the same pattern on the class.  I think just `private` should be enough.  As I don't think we need to create a new instance of the class in order to access the variable to be referenced inside of the class.